src/app/App.tsx

```diff
@@ -26,13 +27,25 @@ const LibraryStackScreen = () => {
     headerBackground: () => <Image style={styles.imageHeader} source={require('./assets/bc_nav_bar.png')} />,
     headerStyle: styles.headerStyle,
     cardStyle: styles.headerCard,
-    headerTitleStyle: styles.headerTitle,
-    headerLeft: () => <LogoutHeader />
+    headerTitleStyle: styles.headerTitle
   };
   return (
     <LibraryStack.Navigator initialRouteName="Library" screenOptions={headerOptions}>
-      <LibraryStack.Screen name="Library" component={Library} />
+      <LibraryStack.Screen
+        name="Library"
+        component={Library}
+        options={{ headerLeft: () => <LogoutHeader />, headerRight: () => <SearchHeader extended={false} /> }}
+      />
       <LibraryStack.Screen name="BookDetail" component={BookDetail} />
+      <LibraryStack.Screen
+        name="LibrarySearch"
+        component={Library}
+        options={{
+          headerTitle: () => <SearchHeader extended={true} />,
+          headerLeft: () => undefined,
+          headerRight: () => undefined
```

Comment:
?????

Reply:
Lol! This is because, I set a custom Headers for all views in stack but, in that two views the headers is different so i must by pass headerLeft and Right for override the default header and set it again when i need it. How i don't want to change my custom header options need to do that.

Reply:
Anyways... i change undefined for null, i think that is more representative.